### PR TITLE
Add Extensions for Params

### PR DESCRIPTION
**Pull request by MaximeRDY**
Hello, 

I've made a PR for this issue : https://github.com/ruby-grape/grape-swagger/issues/784

close 784

### CODE REVIEW

**Comment by dblock** [comment]
We document extensions in README, how does this relate to that? Feels like this needs a documentation update.

**Comment by MaximeRDY** [comment]
Hello @dblock, 
You are right :-) 
I've added the documentation. 
Feel free if you have any remarks

**Comment by dblock** [comment]
LGTM, leaving to @LeFnord or another active maintainer

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 require 'rack'
 require 'rack/test'
-
+require 'pry'
```
Comment (LeFnord) [inline spec/spec_helper.rb:33]: please remove

**Comment by LeFnord** [comment]
thanks @MaximeRDY … only one last thing

**Comment by LeFnord** [comment]
thanks again @MaximeRDY

**Pull request merged**